packages/plugingen/src/commands/generate/index.ts

```diff
@@ -26,6 +26,15 @@ import { PluginActionsGenPhase } from "./phases/plugin-actions/index.js";
 import { PluginGeneratorPhase } from "./phases/plugin-generator/index.js";
 import type { Phase, PhaseInput } from "./types.js";
 
+// Do not reorder
+// In order of preference files are checked
+export const configFiles = [
+  "plugingen.config.ts",
+  "plugingen.config.js",
+  "plugingen.config.mjs",
+  "plugingen.config.mts",
+];
```

Comment:
find config already has this: https://github.com/alchemyplatform/aa-sdk/blob/main/packages/plugingen/src/utils/findConfig.ts

Reply:
yeah I use the same function, but just made the input file names to search for to the function.